log/zerolog/logger.go

```diff
@@ -20,6 +20,7 @@ var levelMap = map[log.Level]zerolog.Level{
 // Logger abstraction based on zerolog.
 type Logger struct {
 	logger *zerolog.Logger
+	level  log.Level
```

Comment:
is there a way to avoid having the level here?

Reply:
Unfortunately we can't because zerolog doesn't expose the logging level.